testdata/example_requests.http

```diff
@@ -0,0 +1,55 @@
+###
+# Login to Easee API
+POST https://api.easee.cloud/api/accounts/login
+Accept: application/json
+Content-Type: application/*+json
+
+{
+  "userName": "markus@futurehome.no",
+  "password": "Easee-ad1"
```

Comment:
Tsk tsk

Reply:
Forgot it's a public repo..

Reply:
Fixed

Reply:
Not sure if this should be a public repo though. Have to ask Bartosz